docs/tests.md

```diff
@@ -62,15 +62,15 @@ fluid.defaults("my.tests.environment", {
 fluid.tests.runTests("my.tests.environment");
 ```
 
-# `gpii.test.pouch.harness`
+# `gpii.test.pouch.environment.persistent`
 
-An instance of `gpii.express` which has an instance of `gpii.pouch` wired into it.  Provides a working HTTP REST API
-equivalent to CouchDB.  Although you will generally use this in combination with the test enviroment above, it can
-also be used on its own. As an example, the file `tests/js/launch-test-harness.js` included with this package can be
-used to launch a standalone test instance of this package for manual QA.
+An extension of `gpii.tests.pouch.environment`, which allows for cases in which data should persist between runs.
```

Comment:
I had to create one at least to test persistence in this package.  I can see moving it into those tests and out of the documentation, if that seems reasonable.